tests/cancel.rs

```diff
@@ -5,9 +5,10 @@ use utils::*;
 fn cancel_paid() {
     let system = init_system();
     let escrow_program = init_escrow(&system);
-    let ft_program = init_ft(&system);
+    let ft_program = gtest::Program::ftoken(WALLET[0] as u64, FT_PROGRAM_ID, &system);
```

Comment:
Publicly import `Program` in `utils/mod.rs` to avoid repetitions of `gtest::`.
```suggestion
    let ft_program = Program::ftoken(WALLET[0] as u64, FT_PROGRAM_ID, &system);
```